src/envs/pybullet_blocks.py

```diff
@@ -452,7 +477,8 @@ def _get_state(self) -> State:
         state_dict = {}
 
         # Get robot state.
-        state_dict[self._robot] = self._pybullet_robot.get_state()
+        state_dict[self._robot] = self._pybullet_robot.get_object_state()
```

Comment:
can we have a better name for this method? "object" seems to mean "things other than robots"

Reply:
get_state() and get_joints()?